core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -94,7 +94,7 @@ contract DesignatedVoting is Withdrawable {
     function retrieveRewards(uint256 roundId, VotingInterface.PendingRequest[] memory toRetrieve)
         public
         onlyRoleHolder(uint(Roles.Voter))
-        returns (FixedPoint.Unsigned memory rewardsIssued)
+        returns (FixedPoint.Unsigned memory)
```

Comment:
Just putting this here because I can't comment on the actual line.

I noticed you didn't remove the named return values from the `Store`. Was there a reason for that?

Reply:
Ye that was 100% a mistake. I've added it now.